scripts/scripts/vaccinations/automations/incremental/jersey.py

```diff
@@ -4,32 +4,62 @@
 from bs4 import BeautifulSoup
 import vaxutils
 
+date = None
```

Comment:
I don’t see why we need a global `date`.
After parsing the table and returning one row off of it, we could enrich the date with `assign`.